inference-engine/src/transformations/include/transformations/op_conversions/convert_gather_v1_to_gather_v7.hpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <transformations_visibility.hpp>
+
+#include <ngraph/pass/graph_rewrite.hpp>
+
+namespace ngraph {
+namespace pass {
+
+class TRANSFORMATIONS_API ConvertGather1ToGather7;
+
+}  // namespace pass
+}  // namespace ngraph
+
+/**
+ * @ingroup ie_transformation_common_api
+ * @brief ConvertGather1ToGather7 coverts v1::Gather into v7::Gather.
```

Comment:
coverts -> converts

Reply:
fixed

Reply:
Don't see that the issue is fixed.

Reply:
Sorry for confusing, now it's fixed for sure